package.json

```diff
@@ -53,6 +56,9 @@
       "features/session_trace": [
         "dist/types/features/session_trace/index.d.ts"
       ],
+      "features/session_replay": [
+        "dist/types/features/session_replay/index.d.ts"
+      ],
```

Comment:
Still missing the typings entry for soft nav but thats not really part of this PR so I understand if you want to do that separately.